Android/app/src/main/java/com/example/brewhaha_android/Controllers/BottomSheetFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.brewhaha_android.Controllers
```

Comment:
Are you going to add the hookups to this file later (ie. for the buttons and stuff)?

Reply:
Yeah

Reply:
Filtering functionality isn't working yet